packages/contracts-bedrock/scripts/DeploySuperchain.s.sol

```diff
@@ -238,7 +245,7 @@ contract DeploySuperchain is Script {
     // This entrypoint is for end-users to deploy from an input file and write to an output file.
     // In this usage, we don't need the input and output contract functionality, so we deploy them
     // here and abstract that architectural detail away from the end user.
-    function run(string memory _infile) public {
+    function run(string memory _infile, string memory _outfile) public {
```

Comment:
What do you think about removing the outfile arg, and deriving the outfile name from the infile name? For example require the input file to be named `*.DeploySuperchain.in.toml` and the output file name is then just `*.DeploySuperchain.out.toml`. Enforcing this removes an input which is nice UX, and helps make sure it's easy to tell what step an input or output file corresponds to.

There cheats like `vm.split` that we can use along with `string.concat` so the string manipulation should be easy

Reply:
Speaking as a potential consumer of this lib, I much prefer being able to specify the outfile. For example in the Docker I'd like to have it overwrite the infile, which I can't do if the filename is automatically generated.

Happy to switch back if you feel really strongly about this, but I like the idea of keeping them separate.

Reply:
Makes sense. And if we do want to enforce a convention we can always add a `run(string)` overload to have both options are available